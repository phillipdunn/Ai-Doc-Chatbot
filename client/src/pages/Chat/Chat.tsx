import React, { FC } from 'react';

interface Props { }

const Chat: FC<Props> = () => {
  return (
    <>
      <div>Chat renders</div>
    </>
  );
};

export default Chat;
