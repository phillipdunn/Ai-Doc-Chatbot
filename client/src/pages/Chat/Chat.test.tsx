import React from 'react';
import { create } from 'react-test-renderer'; // Import the 'create' function instead of 'ReactTestRenderer'
import Chat from './Chat';

describe('Chat', () => {
    it('should render without crashing', () => {
        const component = create(<Chat />);
        expect(component).toBeTruthy();
    });
});
