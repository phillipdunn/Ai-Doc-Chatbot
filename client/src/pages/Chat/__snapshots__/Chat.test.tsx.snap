// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Chat renders 1`] = `
<div>
  Chat renders
</div>
`;
